app/serializers/reviewable_category_expert_suggestion_serializer.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require_dependency 'reviewable_serializer'
+
+class ReviewableCategoryExpertSuggestionSerializer < ReviewableSerializer
+  attributes :endorsed_by, :endorsed_count, :category, :user
+
+  def endorsed_users
+    @endorsed_users ||= CategoryExpertEndorsement
+      .includes(:user)
+      .where(category_id: object.target.category_id, endorsed_user_id: object.target.endorsed_user_id)
+      .map(&:user)
+  end
+
+  def endorsed_by
+    ActiveModel::ArraySerializer.new(
+      endorsed_users,
+      each_serializer: BasicUserSerializer,
+      root: nil,
+    )
+  end
+
+  def endorsed_count
+    endorsed_users.count
+  end
+
+  def category
```

Comment:
Couldn't you use `has_one` instead of this?

Reply:
Since it's `object.target.category`, rather than `object.category`, I don't think that I can?

Reply:
I believe it'll work if you do `has_one :category` then return `object.target.category` from this method (let it handle the serialization)